File: Competitive Programming/Q1_Factorial/Factorial.py
def fact_rec(num):
    if num==1:
        return num
    else:
        return num*fact_rec(num-1)
inp = int(input("number of inputs : "))
for i in range(inp):
    numb = int(input("number : "))
    if numb < 0:
        print("wrong")
    elif numb == 0:
        print("the factorial of the number is : 1")
    else:
        print("the factorial of the number is", fact_rec(numb)) 

File: Competitive Programming/Q1_Factorial/factorial.c
#include<stdio.h>
int factorial(int n) {
    if (n>=1)
        return n*factorial(n-1);
    else
        return 1;
}
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d",&n);
    printf("Factorial of %d = %ld", n, factorial(n));
    return 0;
}

